Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -119,7 +125,8 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ...
    ```
 
-   If you see the warning at the end, add the following lines to your ``>> ~/.bashrc`` (if you are using bash) or ``>> ~/.zshrc`` (if you are using zsh).
+   > [!WARNING]
```

Comment:
These don't seem to be showing up properly in the rich text diff. In future, please do a preview check before submitting for review.